### ISSUE TITLE
`_backoffice.Backoffice.publish` should probably include `reviewer` field or similar 

### ISSUE THREAD

**Issue by jmetz** [bioimage-io/collection#25]
For keeping track of who published a resource.

I can see that in the context of CI-based publishing this would have to be an action input (ie a little more involved), but we could always set the review to `CI` for the time-being for those cases.

Or should a "publish" action by a reviewer **always** invoke a CI call? 
Either way, we probably should start tracking which user kicked-off a CI call, something that isn't done yet IIRC.

**Comment by FynnBe** [comment]
yes, I'll ad that. we can identify the reviewer by their token if publish is executed via dispatch

**Comment by jmetz** [comment]
:+1: also  - same for `request_changes`.

**Comment by jmetz** [comment]
Just added the initial draft of the review service here: https://github.com/bioimage-io/bioimageio-uploader-review
